ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt

```diff
@@ -117,6 +117,11 @@ class MainActivity : ComponentActivity() {
         }
     }
 
+    val permissions = arrayOf(
```

Comment:
plz test getting permissions from manifest.
https://stackoverflow.com/questions/18236801/programmatically-retrieve-permissions-from-manifest-xml-in-android

Reply:
Updated! Thanks.